staging/src/k8s.io/component-base/config/v1alpha1/types.go

```diff
@@ -141,3 +141,18 @@ type VModuleItem struct {
 	// that match the pattern.
 	Verbosity uint32 `json:"verbosity"`
 }
+
+// TracingConfiguration provides versioned configuration for tracing clients.
+type TracingConfiguration struct {
+	// +optional
+	// Endpoint of the collector this component will report traces to.
+	// The syntax is defined in https://github.com/grpc/grpc/blob/master/doc/naming.md.
+	// Default is the otlp grpc default, localhost:4317
+	// The connection is insecure, and does not currently support TLS.
```

Comment:
See comments on the internal type above.

Reply:
updated, ptal, it's now the same in both config & config/v1alpha1 as below:
```go
// TracingConfiguration provides versioned configuration for OpenTelemetry tracing clients.
type TracingConfiguration struct {
    // Endpoint of the collector this component will report traces to.
    // The connection is insecure, and does not currently support TLS.
    // Recommended is unset, and endpoint is the otlp grpc default, localhost:4317.
    // +featureGate=KubeletTracing
    // +optional
    Endpoint *string

    // SamplingRatePerMillion is the number of samples to collect per million spans.
    // Recommended is unset. If unset, sampler respects its parent span's sampling
    // rate, but otherwise never samples.
    // +featureGate=KubeletTracing
    // +optional
    SamplingRatePerMillion *int32
}

```